.npmignore

```diff
@@ -1,16 +1,11 @@
-.github/
-ci/
-coverage/
-examples/
-test/
-**/*.test.ts
-.babelrc
-.editorconfig
-CODE_OF_CONDUCT.md
-CONTRIBUTE.md
-build.js
-.vscode/
-.idea/
-tsconfig-test.json
-jest.config.json
-.npmrc
+# This file is written to be a whitelist instead of a blacklist. Start by
+# ignoring everything, then add back the files we want to be included in the
+# final NPM package.
+*
+
+# And these are the files that are allowed.
+!/LICENSE
+!/README.md
+!/package.json
+!/dist/**/*
+!/types/**/*
```

Comment:
Can you please add back all the directories and files which were being ignored

Reply:
I can add it back, but this file now works like a whitelist instead, so you only have to add what you actually want published (see the * in the beginning for blacklist of everything - then after comes the exceptions). Personally I think that's cleaner/safer.

Let me know if I should still revert.